lib/identity_cache/query_api.rb

```diff
@@ -120,7 +120,14 @@ def check_association_scope(association_name)
         end
       end
 
-      def record_from_coder(coder) #:nodoc:
+      def record_from_coder(coder)
+        return unless coder
+        ActiveSupport::Notifications.instrument('identity_cache.hydration', class: coder[:class]) do
```

Comment:
Actually `hydration` comes from the ORM world, where what's meant is "hydrating" the actual Object from a the "split up representation" in this case, the `Hash`. Where as serialization would be getting that "dehydrated" representation into a `byte[]`

That's why I don't like using serialization as the wording, but if it's "better accepted/known" I don't really care so much either. Whatever the consensus is :)